Example/Prototyper Example/PrototyperExampleApp.swift

```diff
@@ -11,10 +11,10 @@ import Prototyper
 
 
 @main
-struct Prototyper_ExampleApp: App {
+struct PrototyperExampleApp: App {
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            ContentView().prototyper(PrototyperSettings.default)
```

Comment:
I would suggest to add a line break here, same as in the README file.
```suggestion
            ContentView()
                .prototyper(PrototyperSettings.default)
```